### PR TITLE
feat: github actions to publish ecr and docker

### PR DESCRIPTION
**Pull request by krol3**
- Adding github actions to build and release
- Adding github actions to publish in docker and ecr 
- Update the readme with badges
- [Issue-771](https://github.com/aquasecurity/kube-bench/issues/771)

 Please create the followers secrets:
- AWS_ACCESS_KEY_ID
- AWS_SECRET_ACCESS_KEY
- DOCKER_HUB_ACCESS_TOKEN
- DOCKER_HUB_USERNAME

### CODE REVIEW

File: .github/workflow/publish-ecr.yml [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+
+jobs:
+  deploy:
+    name: Publish to Amazon ECR
+    runs-on: ubuntu-18.04
+    steps:
+      - name: Configure AWS credentials
+        uses: aws-actions/configure-aws-credentials@v1
+        with:
+          aws-access-key-id: ${{ secrets.AWS_ACCESS_KEY_ID }}
```
Comment (lizrice) [inline .github/workflow/publish-ecr.yml:15]:
```suggestion
          aws-access-key-id: ${{ secrets.ECR_ACCESS_KEY_ID }}
```

File: .github/workflow/publish-ecr.yml [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+          aws-secret-access-key: ${{ secrets.AWS_SECRET_ACCESS_KEY }}
```
Comment (lizrice) [inline .github/workflow/publish-ecr.yml:16]:
```suggestion
          aws-secret-access-key: ${{ secrets.ECR_SECRET_ACCESS_KEY }}
```

File: .github/workflow/publish-ecr.yml (hunk 1 above)
Comment (lizrice) [reply .github/workflow/publish-ecr.yml:15]: We set up org-wide secrets called ECR_...

**Review by lizrice — APPROVED** [review]
Looks good, thanks! we really want the build action right now so I'm committing it straight away even though there is a small tweak to the secret names

File: .github/workflow/publish-ecr.yml (hunk 1 above)
Comment (krol3) [reply .github/workflow/publish-ecr.yml:15]: Great! thanks

**Pull request merged**